code/Model/VirtualPage.php

```diff
@@ -78,6 +80,11 @@ class VirtualPage extends Page
 
     private static $db = [
         "VersionID" => "Int",
+        'UseProxiedForCanonical' => 'Boolean',
```

Comment:
We can’t add this in a patch release, it’d need to be a separate PR against the `4` branch. It’d also need to be `Boolean(1)`, as `private static $defaults` doesn’t affect existing records